src/plugins/controls/common/options_list/options_list_selections.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { FieldSpec } from '@kbn/data-views-plugin/common';
+
+export type OptionsListSelection = string | number;
+
+export const getSelectionAsFieldType = (field: FieldSpec, key: string): OptionsListSelection => {
```

Comment:
Is this function needed? How about just calling `String(key)` where key needs to be a string

Reply:
It is, yes - we need to be able to convert the key to a number where necessary.